src/components/inputs/datePicker/datePicker.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { DatePicker } from './datePicker.component';
+
+export default {
+  title: 'Components/Inputs/DatePicker',
+  component: DatePicker,
+  parameters: {
+    componentSubtitle: 'Pick a date',
+  },
+} as ComponentMeta<typeof DatePicker>;
+
+export const DatePickers: ComponentStory<typeof DatePicker> = () => {
+  const [value, setValue] = useState(new Date());
+  const handleChange = (newValue: any) => {
+    setValue(newValue);
+  };
+  return (
+    <div className="story__datePicker">
```

Comment:
Remove this if you don't use it. Also the syntax used here comes from [BEM](http://getbem.com/), so the right one would be `story__date-picker`.

Reply:
Also removed this